capresponse/loginResponse.go

```diff
@@ -1,34 +0,0 @@
-//(C) Copyright [2020] Hewlett Packard Enterprise Development LP
```

Comment:
why this file is removed

Reply:
The structs in the file are not being used.